packages/mobile/src/screens/search-results-screen/tabs/ProfilesTab.tsx

```diff
@@ -0,0 +1,25 @@
+import { makeGetSearchArtists } from 'audius-client/src/common/store/pages/search-results/selectors'
+import { isEqual } from 'lodash'
+
+import { CardList } from 'app/components/core'
+import { FollowArtistCard } from 'app/components/signon/FirstFollows'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+
+import { SearchResultsTab } from './SearchResultsTab'
+
+const getSearchUsers = makeGetSearchArtists()
+
+export const ProfilesTab = () => {
+  const users = useSelectorWeb(getSearchUsers, isEqual)
```

Comment:
Not sure if `isEmpty` was intentionally added bc of performance issues. But it might be more performant to rerender every time here, not sure

Reply:
yeah will remove untli we give perf a proper treatment